transports/noise/src/io/handshake.rs

```diff
@@ -44,7 +45,7 @@ use std::io;
 /// Handshake state.
 pub(crate) struct State<T> {
     /// The underlying I/O resource.
-    io: NoiseFramed<T, snow::HandshakeState>,
+    io: asynchronous_codec::Framed<T, Codec<snow::HandshakeState>>,
```

Comment:
```suggestion
    io: Framed<T, Codec<snow::HandshakeState>>,
```

Redundant if we are importing it fully :)

Reply:
sure!